packages/@aws-cdk/cfnspec/spec-source/550_ImageBuilder_Image_Attributes_OutputResources_patch.json

```diff
@@ -0,0 +1,15 @@
+{
+  "ResourceTypes": {
+    "AWS::ImageBuilder::Image": {
+      "patch": {
+        "description": "Removes 'OutputResources' attribute as it specifies an undefined Type 'OutputResources'",
+        "operations": [
+          {
+            "op": "remove",
+            "path": "/Attributes/OutputResources"
+          }
+        ]
```

Comment:
That's pretty radical. The documentation says this:

> OutputResources:
>
> Returns the output resources produced when creating this image, formatted as an array of AMIs.

That does look like an array of strings to me. Could be worth trying to use to confirm.